### PR TITLE
Update golang to active supported version

### PR DESCRIPTION
**Pull request by henry118**
Go v1.19 is EOL. Update Go to active versions.

### CODE REVIEW

File: scripts/release.sh [hunk 1]
```diff
@@ -1,6 +1,7 @@
 #!/usr/bin/env bash
 set -xe
 
+GOLANG="${GOLANG:-golang:1.21-alpine}"
```
Comment (henry118) [inline scripts/release.sh:4]: wondering where is `build.sh` in this script and found #961 :) happy to revert the changes to this file and rebase if that PR is merged.

**Comment by henry118** [comment]
@squeed @s1061123 PTAL

File: scripts/release.sh (hunk 1 above)
Comment (squeed) [reply scripts/release.sh:4]: I'll merge that :-)

**Review by squeed — APPROVED** [review]
